aspnetcore/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Views/Movies/IndexGenreRating.cshtml

```diff
@@ -1,4 +1,5 @@
-﻿@model MvcMovie.Models.MovieGenreViewModel
+﻿#if COMBINED
```

Comment:
Why are you adding this?

Reply:
I added it to make it clear that it is only used if its matching model using `COMBINED` is currently defined.  Otherwise it is hard to see what models are intended to work with what views.    Technically it is not needed, but it was intended to add clarity since we have several in there.   If you think it adds confusion rather than helping I'll take it out.

Reply:
Remove it.